[2024-04-13 19:30:01] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 2680fb919c17c10a [::1]:46960 failed checkpoint posthandshake: Connected to self
[2024-04-13 19:30:01] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn static dial conn 2680fb919c17c10a [::1]:20132 failed checkpoint posthandshake: Connected to self
[2024-04-13 19:30:04] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 19:30:09] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 19:30:14] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 19:30:19] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 19:30:24] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 19:30:29] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 19:30:31] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 2680fb919c17c10a [::1]:37550 failed checkpoint posthandshake: Connected to self
[2024-04-13 19:30:31] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn static dial conn 2680fb919c17c10a [::1]:20132 failed checkpoint posthandshake: Connected to self
[2024-04-13 19:30:34] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 19:30:39] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 19:30:44] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 19:30:49] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 19:30:54] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 19:30:59] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
